ledger-service/metrics/src/main/scala/com/digitalasset/metrics/api/reporters/MetricsReporting.scala

```diff
@@ -53,7 +54,7 @@ final class MetricsReporting[M](
       _ <- Resource(Future.successful(slf4JReporter))(reporter =>
         Future.successful(reporter.report())
       )
-    } yield metrics(registry, openTelemetryMeter)
+    } yield metrics(registry, openTelemetry.getMeter("daml"))
```

Comment:
It's used to avoid conflicts when otel is used by multiple libraries, introducing a namespace for each of them. As the sdk is referenced in multi places in the metrics as daml I would prefer we keep it consistent like that. The namespace for canton would be `canton`